src/main/java/com/sarality/db/query/QueryParser.java

```diff
@@ -0,0 +1,56 @@
+package com.sarality.db.query;
+
+import android.text.TextUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Given a query, reverses the Query Builder operations and extracts the columnlist, operators and arguments
+ *
+ * @author satya@ Satya Puniani
```

Comment:
We should look into formatting things properly somehow

The format here is 

@author satya@ (Satya Puniani)